reproman/support/exceptions.py

```diff
@@ -197,3 +197,22 @@ class SSHAccessDeniedViaAuthKeys(BaseException):
     """
     def __init__(self, user):
         self.msg = "access for user '%s' denied via authorized_keys" % user
+
+
+#
+# Run related exceptions
+#
+class JobError(RuntimeError):
+    """reproman run  execution related error.
```

Comment:
nit-pick: extra space

Reply:
argh - my habit for visual separation of commands from the text. Will do as """`reproman run`"""